src/realm/table.cpp

```diff
@@ -1053,13 +1053,23 @@ bool Table::set_embedded(bool embedded)
         });
 
```

Comment:
Shouldn't we skip all the following checks when migrating from embedded to non-embedded (eg: `set_embedded(false)`)?